data/mods/Magiclysm/Spells/stormshaper.json

```diff
@@ -3,7 +3,7 @@
     "id": "jolt",
     "type": "SPELL",
     "name": "Jolt",
-    "description": "A short ranged fan of elecricity shoots from your fingers.",
+    "description": "A short ranged fan of electricity shoots from your fingers.",
```

Comment:
```suggestion
    "description": "A short-ranged fan of electricity shoots from your fingers.",
```

Reply:
Migic "Light" will cause the detection result of gettext translation to confuse it with the "Light" candle.
I hope to change it to a name.